bin/gssstdio.c

```diff
@@ -496,7 +496,7 @@ read_packet(int fd, gss_buffer_t buf, int timeout, int first)
 		return -2;
 
 	/* We have the complete length */
-	len = ntohl(*(uint32_t *)len_buf);
+	len = ntohl(len);
```

Comment:
The commit comment should be something like `Fix potentially unaligned pointer dereference`.

Reply:
Commit subject lines need to be pithy, yes, but as much as possible they need to describe the bug being fixed (if a bug) so that `git log --oneline` can be helpful.